### PR TITLE
Update Setup index

### PR DESCRIPTION
**Pull request by gocadimic**

### CODE REVIEW

**Review by jpradocueva — APPROVED** [review]
@gocadimic approved and I will merge now.

File: content/en/md2html-setup.md [hunk 1]
```diff
@@ -43,13 +42,30 @@ files:
 - Appendix_L.md
 - Appendix_M.md
 ---
+# Detailed instructions see at https://markdown.standardshub.io/md2html-setup.
```
Comment (jpradocueva) [inline content/en/md2html-setup.md:45]: @gocadimic, I understand your question now. The guideline comment needs to be inside the example. People can copy the example and past them on their new index.yaml file.

File: content/en/md2html-setup.md [hunk 2]
```diff
@@ -43,13 +42,30 @@ files:
...
-* `organizationName`:  represents the name of the organization.
-* `copyrightDate`:         the copyright date is defined by this property.
-* `logo`:                          this allows each document to have its own logo
-* `watermark`:               this property inserts a watermark inside of the PDF. Possible values are: "DRAFT",  "CANDIDATE", "HISTORIC", "DEPRECATED", "PRE-APPROVED", "APPROVED".
+### Properties
+* `title`
+    * Title of the document, as displayed on the front page
+* `status`
+    * Document status, e.g., `Draft`, `Candidate`, `Approved`
+* `version`
```
Comment (jpradocueva) [inline content/en/md2html-setup.md:53]: @gocadimic I have added the deleted properties from Version V1.x. In this way, we can provide an update on each existing properties.

**Review by jpradocueva — APPROVED** [review]
@gocadimic please check the comments and changes and if you agree merge the PR.
Thanks!

File: content/en/md2html-setup.md (hunk 2 above)
Comment (gocadimic) [reply content/en/md2html-setup.md:53]: @jpradocueva, 
I understand, in any case, we have that version for OMA.

File: content/en/md2html-setup.md (hunk 1 above)
Comment (gocadimic) [reply content/en/md2html-setup.md:45]: @jpradocueva,
Ok, I will remove it there.

**Comment by jpradocueva** [comment]
@gocadimic, it appears that this PR has failed. Please do a `git fetch origin` from your side, merge if needed and then push.

**Comment by gocadimic** [comment]
@jpradocueva, everything seems even from my side, I think for this PR you need to do it from your side.

**Pull request merged**